cea/optimization/preprocessing/decentralized_buildings.py

```diff
@@ -95,32 +95,32 @@ def calc_new_load(mdot, TsupDH, Tret, gv):
 
             Qgas = Qload[hour] / BoilerEff
 
-            result[0][4] += gv.NG_PRICE * Qgas # CHF
-            result[0][5] += gv.NG_BACKUPBOILER_TO_CO2_STD * Qgas * 3600E-6 # kgCO2
-            result[0][6] += gv.NG_BACKUPBOILER_TO_OIL_STD * Qgas * 3600E-6 # MJ-oil-eq
+            result[0][4] += prices.NG_PRICE * Qgas # CHF
+            result[0][5] += optimization_constants.NG_BACKUPBOILER_TO_CO2_STD * Qgas * 3600E-6 # kgCO2
+            result[0][6] += optimization_constants.NG_BACKUPBOILER_TO_OIL_STD * Qgas * 3600E-6 # MJ-oil-eq
             resourcesRes[0][0] += Qload[hour]
 
-            if gv.DiscBioGasFlag == 1:
-                result[0][4] += gv.BG_PRICE * Qgas # CHF
-                result[0][5] += gv.BG_BACKUPBOILER_TO_CO2_STD * Qgas * 3600E-6 # kgCO2
-                result[0][6] += gv.BG_BACKUPBOILER_TO_OIL_STD * Qgas * 3600E-6 # MJ-oil-eq
+            if optimization_constants.DiscBioGasFlag == 1:
+                result[0][4] += prices.BG_PRICE * Qgas # CHF
+                result[0][5] += optimization_constants.BG_BACKUPBOILER_TO_CO2_STD * Qgas * 3600E-6 # kgCO2
+                result[0][6] += optimization_constants.BG_BACKUPBOILER_TO_OIL_STD * Qgas * 3600E-6 # MJ-oil-eq
 
             # Boiler BG
-            result[1][4] += gv.BG_PRICE * Qgas # CHF
-            result[1][5] += gv.BG_BACKUPBOILER_TO_CO2_STD * Qgas * 3600E-6 # kgCO2
-            result[1][6] += gv.BG_BACKUPBOILER_TO_OIL_STD * Qgas * 3600E-6 # MJ-oil-eq
+            result[1][4] += prices.BG_PRICE * Qgas # CHF
+            result[1][5] += optimization_constants.BG_BACKUPBOILER_TO_CO2_STD * Qgas * 3600E-6 # kgCO2
+            result[1][6] += optimization_constants.BG_BACKUPBOILER_TO_OIL_STD * Qgas * 3600E-6 # MJ-oil-eq
             resourcesRes[1][1] += Qload[hour]
 
             # FC
-            (FC_Effel, FC_Effth) = FC.calc_eta_FC(Qload[hour], Qnom, 1, "B")
+            (FC_Effel, FC_Effth) = FC.calc_eta_FC(Qload[hour], Qnom, optimization_constants, 1, "B")
             Qgas = Qload[hour] / (FC_Effth+FC_Effel)
             Qelec = Qgas * FC_Effel
 
-            result[2][4] += gv.NG_PRICE * Qgas - gv.ELEC_PRICE * Qelec # CHF, extra electricity sold to grid
-            result[2][5] += 0.0874 * Qgas * 3600E-6 + 773 * 0.45 * Qelec * 1E-6 - gv.EL_TO_CO2 * Qelec * 3600E-6 # kgCO2
+            result[2][4] += prices.NG_PRICE * Qgas - prices.ELEC_PRICE * Qelec # CHF, extra electricity sold to grid
+            result[2][5] += 0.0874 * Qgas * 3600E-6 + 773 * 0.45 * Qelec * 1E-6 - optimization_constants.EL_TO_CO2 * Qelec * 3600E-6 # kgCO2
             # Bloom box emissions within the FC: 773 lbs / MWh_el (and 1 lbs = 0.45 kg)
             # http://www.carbonlighthouse.com/2011/09/16/bloom-box/
-            result[2][6] += 1.51 * Qgas * 3600E-6 - gv.EL_TO_OIL_EQ * Qelec * 3600E-6 # MJ-oil-eq
+            result[2][6] += 1.51 * Qgas * 3600E-6 - optimization_constants.EL_TO_OIL_EQ * Qelec * 3600E-6 # MJ-oil-eq
```

Comment:
what does `3600E-6` mean? another magic number that could be replaced by a constant at the top of the file... the kind of constant that would make reading the line easier to understand. is it a conversion factor?